app/app.go

```diff
@@ -196,6 +196,11 @@ var (
 		emissionsModuleTypes.UndistributedObserverRewardsPool: nil,
 		emissionsModuleTypes.UndistributedTssRewardsPool:      nil,
 	}
+disa
+	// module accounts that are allowed to receive tokens
```

Comment:
NIT :  module accounts that are not allowed to receive tokens

Reply:
done